botocore/client.py

```diff
@@ -606,8 +608,21 @@ def _create_endpoint(
             resolved, region_name, endpoint_url
         )
         if endpoint_url is None:
-            # Use the sslCommonName over the hostname for Python 2.6 compat.
-            hostname = resolved.get('sslCommonName', resolved.get('hostname'))
+            sslCommonName = resolved.get('sslCommonName')
+            hostname = resolved.get('hostname')
+            disable = ensure_boolean(
```

Comment:
nit; looking at this again, maybe `is_disabled` is more consistent with our other booleans.